src/main/webapp/app/shared/info-panel/info-panel.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { InfoPanelComponent } from './info-panel.component';
+
+@NgModule({
+    imports: [ArtemisSharedModule],
+    declarations: [InfoPanelComponent],
+    exports: [InfoPanelComponent],
```

Comment:
Same here, if it is indended to be used everywhere in the application it might as well become part of the shared module

Reply:
yes, done 👍